contributors/help-wanted.md

```diff
@@ -0,0 +1,138 @@
+  new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`", with
+  expected validations called out.
+
+- **Goldilocks priority**
+  The priority should not be so high that a core contributor should do it, but not too low that it
+  isn't useful enough for a core contributor to spend time reviewing it, answering
+  questions, helping get it into a release, etc.
+
+- **Up-To-Date**
+  Often these issues become obsolete and have already been completed, are no longer
+  desired, no longer make sense, or have changed priority or difficulty.
+
+<!-- TODO find a good one
+A good example of a Help Wanted issue description can be found here: [kubernetes/test-infra#21356 (comment)](https://github.com/kubernetes/test-infra/issues/21356#issuecomment-799972711).
+-->
+
+These commands can be used with GitHub issues to manage the `help wanted` label:
+
+- `/help` : Adds the `help wanted` label to an issue.
+- `/remove-help` : Removes the `help wanted` label from an issue. If the
+  `good first issue` label is present, it is removed as well. 
+
+## Good First Issue
+
+Items marked with the `good first issue` label are intended for _first-time
+contributors_. It indicates that members will keep an eye out for these pull
+requests and shepherd it through our processes.
+
+**New contributors should not be left to find an approver, ping for reviews,
+decipher prow commands, or identify that their build failed due to a flake.**
+It is important to make new contributors feel welcome and valued. We should assure them that they
+will have an extra level of help with their first contribution.
+
+After a contributor has successfully completed one or two `good first issue` items, they
+should be ready to move on to `help wanted` items.
+
+All `good first issue` items need to follow the guidelines for `help wanted`
+items in addition to meeting the following criteria:
+
+- **No Barrier to Entry**
+  The task is something that a new contributor can tackle without advanced
+  setup or domain knowledge.
+
+- **Solution Explained**
+  The recommended solution is clearly described in the issue.
+
+- **Provides Context**
+  If background knowledge is required, this should be explicitly mentioned and a
+  list of suggested readings included.
+
+- **Gives Examples**
+  Link to examples of similar implementations so new contributors have a
+  reference guide for their changes.
+
+- **Identifies Relevant Code**
+  The relevant code and tests to be changed should be linked in the issue.
+
+- **Ready to Test**
+  There should be existing tests that can be modified, or existing test cases
+  fit to be copied. If the area of code doesn't have tests, before labeling the
+  issue, add a test fixture. This prep often makes a great `help wanted` task!
+
+<!-- TODO find a good one
+A good example of a `good first issue` description can be found here: [kubernetes/kubernetes#68231](https://github.com/kubernetes/kubernetes/issues/68231).
+-->
```

Comment:
Replace with KubeVirt specific issue?

Reply:
This is enclosed inside html (or rather xml) multiline comment tags, that's why the `TODO` marker is still there.

Reply:
But yes, I am going to remove this.